recipe/meta.yaml

```diff
@@ -1,52 +1,47 @@
-{% set name = "Fabric" %}
-{% set version = "1.14.0" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "81572002a562b26df657ca10d06b5bcfc6fd56b53374037392c83d4a2158208f" %}
-{% set build = 1000 %}
+{% set name = "fabric" %}
+{% set version = "2.5.0" %}
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 24842d7d51556adcabd885ac3cf5e1df73fc622a1708bf3667bf5927576cdfa6
 
 build:
-  skip: True  # [py3k]
+  number: 0
+  noarch: python
   entry_points:
-    - fab = fabric.main:main
-  number: {{ build }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+    - fab = fabric.main:program.run
+  script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
-  build:
+  host:
+    - pip
     - python
-    - setuptools
-
   run:
+    - invoke >=1.3,<2.0
+    - paramiko >=2.4
     - python
-    - paramiko >=1.10,<3.0
 
 test:
   imports:
     - fabric
-    - fabric.contrib
-
+    - fabric.testing
   commands:
     - fab --help
 
 about:
-  home: http://fabfile.org
-  license_file: LICENSE
-  license: BSD 2-Clause
+  home: "http://fabfile.org"
+  license: BSD
   license_family: BSD
-  summary: 'Fabric is a simple, Pythonic tool for remote execution and deployment.'
+  license_file: LICENSE
+  summary: "High level SSH command execution"
```

Comment:
I think the original summary was better :wink:

Reply:
Agreed! My bad;; that was unintentional as I built it from a new `skeleton` file.